dozer-tests/src/tests/e2e/basic_sql.rs

```diff
@@ -0,0 +1,62 @@
+use std::{
+    sync::{atomic::AtomicBool, Arc},
+    thread,
+};
+
+use dozer_api::actix_web::rt::Runtime;
+use dozer_orchestrator::{simple::SimpleOrchestrator as Dozer, Orchestrator};
+use dozer_types::grpc_types::common::{
+    common_grpc_service_client::CommonGrpcServiceClient, QueryRequest,
+};
+use dozer_types::{models::app_config::Config, serde_yaml};
+use tempdir::TempDir;
+fn get_config() -> Config {
+    let path = TempDir::new("tests").unwrap();
```

Comment:
Does it work? I thought `tempdir` deletes the temporary directory on `TempDir::drop`.

Reply:
Seems to work.